libs/libcommon/include/common/Sleep.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <cstdint>
+
+void SleepForNanoseconds(uint64_t nanoseconds);
```

Comment:
Missing comment.

Reply:
Detailed description is added in `Sleep.cpp`. Function interface is quite straightforward, maybe we could leave it like this?

Reply:
Comment is about motivation of this function.
It should answer a question "why there is need for a separate sleep function".